pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -137,6 +147,42 @@ func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder
 	return resourceapply.ApplyConfigMap(client, recorder, configMap)
 }
 
+func checkForFeatureGates(featureGateLister configlistersv1.FeaturesLister) (map[string]bool, error) {
+	featureGateListConfig, err := featureGateLister.Get("cluster")
+	var featureGates = make(map[string]bool)
+	if apierrors.IsNotFound(err) {
+		glog.Infof("features.config.openshift.io/cluster: not found, so return empty feature gates")
+		return nil, err
+	}
+	var enabledFeatureSets, disabledFeatureSets []string
+	currentFeatureSetConfig := featureGateListConfig.Spec.FeatureSet
+	switch {
+	case currentFeatureSetConfig == "" || currentFeatureSetConfig == v1.Default:
```

Comment:
@rphillips - Is this how feature gates are intended to be used by caller or did I miss something?

Reply:
v1.Default === "" so you can just pull the FeatureSet out with:

featureSet, ok := v1.FeatureSets[currentFeatureSetConfig]

No need for the switch.

Reply:
Ya, my question was more on the lines of the usage of `currentFeatureSetConfig` but I think I am doing it right. Thank you :). Will remove switch..